src/Makefile-env.am

```diff
@@ -256,8 +256,6 @@ if ENABLE_COVERAGE
 EXTRALIBS += -lgcov
 endif # ENABLE_COVERAGE
 
-LIBCOMMON += -luuid
```

Comment:
On Sat, May 28, 2016 at 8:20 PM, Willem Jan Withagen <
notifications@github.com> wrote:

> In src/Makefile-env.am
> <https://github.com/ceph/ceph/pull/9371#discussion_r64988297>:
>
> > @@ -256,8 +256,6 @@ if ENABLE_COVERAGE
> >  EXTRALIBS += -lgcov
> >  endif # ENABLE_COVERAGE
> >
> > -LIBCOMMON += -luuid
>
> @ukernel <https://github.com/ukernel> Not sure if Freebsd can link
> correctly with uuid
>

This commit exists in my old wip-osx-freebsd branch. I did test it.


> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ceph/ceph/pull/9371/files/d63fe5b93266824212264712ff71ca09968d273e..7425afb261d17a8e1d75c47c63b29739fd5d4f8c#r64988297>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACpZr8J2HO51NvpTMJ48531ssO2jgyJdks5qGDL1gaJpZM4IokCx>
> .
>